src/pages/myGeotab/apiReference/objects.tsx

```diff
@@ -14,15 +15,15 @@ request.send();
 let xml: any = request.responseXML;
 
 const pageTitle: PageTitleProps = {
-    "title": "Objects",
-    "breadCrumbItems": ["MYG", "API Reference", "Objects"]
+    "title": "OBJECTS",
+    "breadCrumbItems": ["MYG", "API RERFENCE", "OBJECTS"]
 };
 
 const pageSections: TableOfContentsItem[] = [
 
 ];
 
-const objects = Object.entries(myGParser(xml, 'object', ['T:Geotab.Checkmate.ObjectModel', 'P:Geotab.Checkmate.ObjectModel']));
+const objects = Object.entries(myGParser(xml, 'object', ['T:Geotab.Checkmate.ObjectModel', 'T:Geotab.Checkmate.API','P:Geotab.Checkmate.ObjectModel', 'M:Geotab.Checkmate.API.#ctor', 'F:Geotab.Checkmate.ObjectModel'])).sort(sortAlphabetical);
 const objectItems = objects.map((objectDetails: any) => {
```

Comment:
`any` should be replaced with the actual type here. objects & objectItems should also have a type specified.